source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -0,0 +1,138 @@
+ - Cloud > "Tips & Next Steps" should not show "Explore channels" section for guest users.
+ - System Roles shows License and Environment as possible permissions but are always hidden in Cloud
+
+## Release 2020-12-03
+
+### Bug Fixes
+  - Disabled the xmlsec1-based SAML library in favor of the re-enabled and improved SAML library.
+
+## Release 2020-11-24
+
+### Compatibility
+ - PostgreSQL ended long-term support for [version 9.4 in February 2020](https://www.postgresql.org/support/versioning). Mattermost is officially supporting PostgreSQL version 10 with v5.26 release as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. We plan on fully deprecating PostgreSQL 9.4 and all 9.x versions in our v5.30 release (December 16, 2020). Please follow the instructions under the Upgrading Section within [the PostgreSQL documentation](https://www.postgresql.org/support/versioning/).
+
+### Highlights
+
+#### Modify new Admin roles permissions in the System Console (Beta)
+ - Added a user interface for the existing new system roles functionality including a new permission to read and write system_roles.
+
+#### Pre-package and enable incident management v1.1.1
+
+### Improvements
+
+#### Cloud-specific
+ - Slack imports are no longer allowed in Cloud installations.
+ - Mattermost cloud instances with delinquent subscriptions will be sent emails on a certain cadence, in certain cases:
+    - A check happens twice per day to see if an installation is over the user limit
+    - If that installation is over the user limit, they are marked for notification during the next billing cycle
+    - If that installation is still over the limit 7, 14, 30, and 90 days into the next billing cycle, they will be sent emails with various wording to get them to add a payment method and upgrade their cloud billing 
+    - If that installation is no longer over the limit 7, 14, and 30 days into the next billing cycle, we will "forgive" them, and not send any emails for that cycle. 
+    - After the installation has gone over the limit, each time the job runs, it checks Stripe to check the state of the subscription. If it's flipped from "past_due" to active, it will cancel any emails for that cycle. Otherwise, it will continue on.
+
+#### User Interface
+ - @-autocomplete results are now prioritized based on recency and thread activity.
+ - File attachments below size of 10 now show fractions.
+ - The formatting of the channel header change message was improved.
+ - Team invite workflow now shows BOT tags when the search returns Bot users.
+ - Added the ability to zoom in/out of PDF files.
+ - Added support for 16x16 base64 encoded mini images to use with progressive rendering.
+
+#### Notifications
+ - Channel-wide mentions are now automatically disabled when a user mutes a channel.
+
+#### Command Line Interface (CLI)
```

Comment:
We probably do not need this for external-facing docs since users cannot access this fucntionality.

Reply:
Removed this and the others below.